docs/notebooks/datasets_modis.ipynb

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
Empty notebook - shall we make a new issue to add this?

Reply:
It is filled with examples now.